tests/testsuite.src/run_ml.at

```diff
@@ -117,6 +124,14 @@ AT_DATA([prog.cob], [
                DISPLAY 'Test 10 failed: ' FUNCTION TRIM (out)
            END-IF
            .
+
```

Comment:
Please copy this testcase adding a new one XML GENERATE WITH ATTRIBUTES, this helps to easier spot later where issues are and to stay focused when debugging.
Also you can add a comment referencing the original bug report :-)